src/expandableCalendar/index.tsx

```diff
@@ -403,6 +405,15 @@ const ExpandableCalendar = (props: ExpandableCalendarProps) => {
     }
   };
 
+  const closeCalendar = () => {
```

Comment:
You need to wrap with useCallback 
Unless we can move `isOpen` to be a ref. 

I noticed you're using isOpen in a lot of methods, but not sure if it actually affects the UI. 
If it's just a flag, do you think we can use a ref for it?

Reply:
What do you mean `isOpen` doesn't affect the UI? Many styles depend on this check...

Reply:
right, so nvm